### ISSUE TITLE
[Suggestion] Set name of active consolidated plugin based on chosen plugin inside of consolidated.

### ISSUE THREAD

**Issue by DivannKokos** [baconpaul/airwin2rack#156]
Hi, I actually don't know if it is possible at all, but it would be pretty cool if plugin could somehow rename its active instance, based of what plugin running inside of consolidated.
Example, If I put a lot of Consolidated plugins on mix, especially several on each channel, then I need to click on each of it, to found out what plugin is there. Since every plugin shows up on channel as Consolidated.

Since it would probably rely on DAW, and every DAW functions differently, I doubt that is even possible, but if it is...
![{E5D2CDBA-3960-48C3-8E63-1B7A76DF9360}](https://github.com/user-attachments/assets/ab7d6ad5-d6e8-4010-a5bb-5e9280904bc4)
Example DAW on picture: Reaper 7
I could rename every instance each time manually, buuttt.... I better then simply using individual plugins, rather than consolidated.

**Comment by baconpaul** [comment]
Yeah alas the daw APIs do not have a mechanism to rename the plugin. You are not the first to ask but I can’t figure out how to do this

**Comment by baconpaul** [comment]
closing since there is no daw api for this. :(
